### ISSUE TITLE
Selecting a relation item needs three clicks

### ISSUE THREAD

**Issue by flayks** [directus/directus#9274]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

When clicking on a relational item –here a M2O-, it needs three clicks on the name or any other part to be selected, when only one is necessary when clicking directly on the checkbox.

This might be a feature but it looks like a bug, also it would make more sense if only one click would select the item.

### To Reproduce

Click three times on a relational item, but only once on the checkbox

https://user-images.githubusercontent.com/273716/139579893-968d45ca-f72e-48aa-87a1-51e648504b68.mp4

### What version of Directus are you using?

9.0.0-rc.100

### What version of Node.js are you using?

16.13.0

### What database are you using?

MySQL

### What browser are you using?

Brave

### What operating system are you using?

macOS Monterey (12)

### How are you deploying Directus?

locally

**Comment by azrikahar** [comment]
Duplicate of #8019.
